third_party/terraform/website/docs/r/container_cluster.html.markdown

```diff
@@ -92,6 +92,10 @@ output "cluster_ca_certificate" {
 * `cluster_ipv4_cidr` - (Optional) The IP address range of the kubernetes pods in
     this cluster. Default is an automatically assigned CIDR.
 
+* `cluster_autoscaling` - (Optional) Configuration for cluster autoscaling, as described
```

Comment:
Can you add a `, Beta` link like in #914?

Reply:
Done.